src/main/resources/db/migration/V202409111400__add_owner_id_column_to_pets.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pets
+ADD COLUMN IF NOT EXISTS owner_id UUID
```

Comment:
Remove from pr

Reply:
i dont know why is it in here, its already in master